server/models/user.js

```diff
@@ -6,6 +6,11 @@ module.exports = function(sequelize, DataTypes) {
     passwordHash: DataTypes.STRING
   }, {
     classMethods: {
```

Comment:
Hey Leo, I think we can change these classMethods to be instanceMethods, except for the "associate", that one is OK as a classMethod